### ISSUE TITLE
Fomod installer doesn't seem to work

### ISSUE THREAD

**Issue by rabidrivas** [limo-app/limo#10]
Mods with fomod installers such as [ASLAL](https://www.nexusmods.com/skyrimspecialedition/mods/272?tab=description) always default to the simple installer and if forced to use the Fomod installer this happens:

![image](https://github.com/user-attachments/assets/455bd39c-1d37-48bf-8bb8-9afde237c504)

**Comment by limo-app** [comment]
The mod you mentioned does not support FOMOD.
Here is a short explanation for this: Let's say you have created a mod that contains a new helmet and want to share it with others. Since not everybody has the same tastes, you offer two color variants: Silver and gold. FOMOD allows you to create a mod that lets others choose which color variant they want to install. For this you have to create a folder called fomod that contains a file named ModuleConfig.xml which defines which files are to be installed when a users selects a specific option. This means only mods that contain a fomod folder with that file inside actually support the FOMOD installer.

TL;DR: A mod must specifically support FOMOD and not every mod does. The installer will automatically detect if it does and will select the appropriate installer for you. You should almost never have to manually select an installer.

**Comment by limo-app** [comment]
Even though the installer will always do this for you anyways, you can check whether a mod supporst FOMOD by clicking on "Preview file contents" on the NexusMods page. If the list does not contain a fomod directory than contains a ModuleConfig.xml file, the mod does not support FOMOD.
